plugins/modules/system/sap_task_list_execute.py

```diff
@@ -0,0 +1,440 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sap_task_list_execute
+short_description: Perform SAP Task list execution.
+version_added: "3.5.0"
+description:
+  - The C(sap_task_list_execute) module depends on C(pyrfc) Python library (version 2.4.0 and upwards).
+    Depending on distribution you are using, you may need to install additional packages to
+    have these available.
+  - Tasks in the task list which requires manual activities will be confirmed automatically.
+  - This module will use the RFC Package C(STC_TM_API).
+
+options:
+  state:
+    description:
+      - If the state C(available_params) is provided the possible parameters are shown for each task.
+        The returned values are in the necessary format for C(task_parameters).
+      - The full list of parameters is exported to stdout in json format.
```

Comment:
```suggestion
```
This depends on the user's callback plugin. If they use the `yaml` callback plugin, they will not see `yaml`. Also they won't see the output when they don't use verbose mode.